### PR TITLE
Backfill perf stats

### PR DESCRIPTION
**Pull request by leohemsted**
to run this on a server:`cf run-task notify-api "python application.py backfill_processing_time -s='2017-07-01' -e='2017-09-01'"`

### CODE REVIEW

File: app/commands.py [hunk 1]
```diff
@@ -167,33 +169,62 @@ def link_inbound_numbers_to_service(self):
...
+        )
+        print("Finished populating data for {} for service id {}".format(month, str(service_id)))
+        print('SMS: {}'.format(sms_res.monthly_totals))
+        print('Email: {}'.format(email_res.monthly_totals))
+
+
+class BackfillProcessingTime(Command):
+    option_list = (
+        Option('-s', '--start_date', dest='start_date', help="Date (%Y-%m-%d) start date inclusive"),
+        Option('-e', '--end_date', dest='end_date', help="Date (%Y-%m-%d) end date inclusive"),
```
Comment (imdadahad) [inline app/commands.py]: It's not inclusive of the end date right

Comment (leohemsted) [reply app/commands.py]: this function is. I call `delta.days + 1` on L218 to specifically make it inclusive.

Comment (imdadahad) [reply app/commands.py]: apologies, missed the commit message.

**Comment by leohemsted** [comment]
good point 👍

**Pull request merged**